branch-and-stage.sh

```diff
@@ -49,3 +67,20 @@ else
   # git add the removed files; hopefully no yaml pollution
   git add --all -fv .
 fi
+
+#Copy from render dir to branch
+for row in $(echo "${pathsToCopy}" | jq -c '.[]'); do
```

Comment:
if you see any chance to merge this as one function it would be great as it has almost the same code. but as I said in another comment probably we can skip one of them

Reply:
extracted into one function